src/utils/bootstrap/bootstrap.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@nestjs/common';
+import { EcoverseService } from 'src/domain/ecoverse/ecoverse.service';
+import { UserInput } from 'src/domain/user/user.dto';
+import { UserService } from 'src/domain/user/user.service';
+
+@Injectable()
+export class BootstrapService {
+  constructor(
+    private ecoverseService: EcoverseService,
+    private userService: UserService
+  ) {}
+
+  async boostrapEcoverse() {
+    this.ecoverseService.getEcoverse();
+    const ctAdminDto = new UserInput();
+    ctAdminDto.name = 'ctAdmin';
+    ctAdminDto.email = 'admin@cherrytwist.org';
+    ctAdminDto.lastName = 'admin';
```

Comment:
Needs to handle the case where the user already exists and also if the user is already in the admins group. It is a failsafe to make sure this user is there.

Also, need to take the setting of the admin role into the context of authorisation design - how to set the admin profile details? For later...

Reply:
I saw the failure and addressed it.